packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -180,40 +181,43 @@ export class Facepile extends BaseComponent<IFacepileProps, {}> {
           title={ personaNames }
           size={ personaSize }
           onRenderInitials={ this._renderInitialsNotPictured(numPersonasNotPictured) }
+          initialsColor={ PersonaInitialsColor.overflowButton }
```

Comment:
The color for `PersonaInitialsColor.overflowButton` is not defined (see other comments below). Is it supposed to be `$ms-color-initials-overflowButton:   #EAEAEA;`? The bg-color of `ms-Facepile-descriptiveOverflowButton` is also `#EAEAEA`